### PR TITLE
Deployment 2021-1.0+ updates

### PR DESCRIPTION
**Pull request by mrakitin**
Deployment tasks: #10.

### CODE REVIEW

**Comment by mrakitin** [comment]
@mfuchsbnl, the merge conflicts have been resolved. I also updated the CI configuration so that it runs the tests for both `collection-2021-1.0` and `collection-2021-1.2` (we don't test the old `collection-2019-3.0-fmx` though, but hopefully it won't be necessary).

**Comment by mrakitin** [comment]
Converted it back to "Draft", so that it's not accidentally merged before the shutdown in August 2021.

File: startup/00-startup.py [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+    # print(f'{print_now()}: waiting for {self.name} to connect within {timeout:.4f} s...')
+    start = time.time()
+    self._ensure_connected(self._read_pv, self._write_pv, timeout=timeout)
+    # print(f'{print_now()}: waited for {self.name} to connect for {time.time() - start:.4f} s.')
+
+EpicsSignalBase.wait_for_connection = wait_for_connection_base
+EpicsSignal.wait_for_connection = wait_for_connection
+###############################################################################
+
+from ophyd.signal import EpicsSignalBase
```
Comment (RobertSchaffer1) [inline startup/00-startup.py:37]: Already imported above.

Comment (RobertSchaffer1) [reply startup/00-startup.py:37]: Although I'm not sure, as it looks like you've overwritten part of it?

Comment (mrakitin) [reply startup/00-startup.py:37]: @RobertSchaffer1, the block between `####... ####` separators is a copy-paste chunk of code that we use to fix the issue with the configured timeouts in the ophyd v1.6.0, that was fixed in v1.6.1 (see https://github.com/bluesky/ophyd/pull/959 for the fix).

**Review by mskinner5278 — APPROVED** [review]

**Review by RobertSchaffer1 — APPROVED** [review]

**Pull request merged**